### DISCUSSION TITLE
変更内容のメモ用ファイルの廃止

### DISCUSSION THREAD

**Discussion by niftg** [ponjbogri/jvs_ja#9] [Ideas]
辞書データの変更内容を別途書き留めるファイル(`cenba_notci*`)を当リポジトリから除外し、明確にXMLエクスポートのみの更新に一本化しようと存じます。

理由としては、現状ほとんど活用されてない点、大量の変更を一度に記録するのには向かない点、版同士の機械的な比較など[^b]で一部代替可能な点が挙げられます。

変更点の詳細をピックアップして書き留める別の場所としては、このGitHubリポジトリに附随のWikiなどを想定して居ります。

異論が特に無ければ、この先のコミットで当該メモファイルを削除し、必要に応じて附属Wikiなどに転記するつもりです。

[^b]: https://github.com/ponjbogri/jvs_ja/discussions/8 はこの目的で為されました

**Comment by qq542vev** [comment]
これは意見ではなくコメントです。当方も幾つかの単語を編集したのですが、非常に細かい修整だったため、報告の必要があるか否かについて迷ってしまい、報告がだいぶ遅れてしまいました。u'u

具体的な変更点には次のようになります。

https://pastebin.com/J55mFDXx

**Reply by niftg** [reply]
> 非常に細かい修整だったため、報告の必要があるか否かについて迷ってしまい、

この点もまさにそれなんですよね。各変更点の特筆性の有無をその都度判断するのはやはり不要な手間ですし、どこかで恣意的にならざるを得ませんし。

当初はリポジトリ内に別記しておいた方が何かしら後で役立つのではと感じたのですが、運用してみて不都合な点が見えました。
